tests/test_api_sources.py

```diff
@@ -236,19 +253,59 @@ def test_sources_cru_existing(session):
     with open(m_file_name, "r") as json_file:
         meta = json.load(json_file)
     # create_source happy case
-    session.create_source(SOURCE_ID, meta)
+    session.create_source(SOURCE_ID_WAVEFORM, meta)
     # list_source happy case
-    src = session.get_source(SOURCE_ID)
-    assert src["source_id"] == SOURCE_ID
+    src = session.get_source(SOURCE_ID_WAVEFORM)
+    assert src["source_id"] == SOURCE_ID_WAVEFORM
     assert src["meta"] == meta
 
     # create_source again (409 ignored)
-    session.create_source(SOURCE_ID, meta)
+    session.create_source(SOURCE_ID_WAVEFORM, meta)
 
     # create_source again (409 not ignored)
     session.do_not_raise = []
     with pytest.raises(HTTPError):
-        session.create_source(SOURCE_ID, meta)
+        session.create_source(SOURCE_ID_WAVEFORM, meta)
+
+
+def test_tabular_sources(session):
+    columns = tabular_df.columns.tolist()
+    columns.remove("timestamp")
+    meta = {"extra": "information"}
+    session.create_tabular_source(SOURCE_ID_TABULAR, meta, columns)
+    src = session.get_source(SOURCE_ID_TABULAR)
+    assert src["source_id"] == SOURCE_ID_TABULAR
+    assert src["meta"] == meta
+    assert src["properties"]["columns"] == columns
+
+
+def test_tabular_measurements_float_timestamps(session):
+    with pytest.raises(HTTPError) as exc:
+        session.create_tabular_measurement(SOURCE_ID_TABULAR, tabular_dict)
+    assert exc.value.response.status_code == 422
+
+
+def test_tabular_measurements(session):
+    columns = tabular_df.columns.tolist()
```

Comment:
Why don't you use a dataset with int timestamps?

Reply:
I forgot to change after enforcing int timestamps in vibium-cloud. I will fix this

Reply:
Or no... I remember now. The dataset had timestamps spaced by exactly 1, so I just removed the decimal. I will change to another dataset to make it simpler